tests/dag_processing/test_processor.py

```diff
@@ -85,9 +78,7 @@ class TestDagFileProcessor:
     @staticmethod
     def clean_db():
         clear_db_runs()
-        clear_db_pools()
         clear_db_dags()
-        clear_db_import_errors()
```

Comment:
These two aren't needed anymore as the tests that assert these don't live in here anymore - either in dagbag or processor tests